x-pack/plugin/esql/qa/server/build.gradle

```diff
@@ -19,7 +19,7 @@ subprojects {
   }
 
 
-  if (project.name != 'security' && project.name != 'mixed-cluster' ) {
+  if (project.name != 'security' && project.name != 'mixed-cluster' && project.name != 'multi-clusters') {
```

Comment:
I don't think we should be adding _more_ projects that use legacy tests. In fact, I thought we actually have a check in the build that fails if we add more projects using this testing. If it's not an enormous effort, we should aim to use the new testing framework here. In fact, multi-cluster testing is much simpler that way.

Reply:
I think this can be removed now.

Reply:
We need to keep this to avoid importing legacy stuff to the new multi-cluster QA module. I will have a follow-up PR to migrate other ESQL QA modules to the new test framework.